src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+const copyColumn = (table, sourceColumnIndex, targetColumnIndex) => {
+  const rows = table.rows;
+
+  for (let i = 0; i < rows.length; i++) {
+    const sourceCell = rows[i].cells[sourceColumnIndex];
+    const newCell = sourceCell.cloneNode(true);
+
+    rows[i].insertBefore(newCell, rows[i].cells[targetColumnIndex]);
+  }
+};
+
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const tfoot = document.querySelector('tfoot');
+
+copyColumn(thead, 1, 4);
+copyColumn(tbody, 1, 4);
+copyColumn(tfoot, 1, 4);
```

Comment:
You can simplify your code like this
```suggestion
const table = document.querySelector('table');

for (const row of table.rows) {
  const copyColumn = row.cells[1].cloneNode(true);
  
  row.lastElementChild.before(copyColumn);
}
```